Typo in the docs

### General information

  * Minishift version: v1.33.0+3b0d08c9
  * OS: Linux
  * Hypervisor: KVM

### Steps to reproduce

  1. Go to: https://docs.okd.io/latest/minishift/contributing/developing.html#running-integration-tests
  2. First sentence assumes that there's a command `make build`, so run it.

### Expected
Minishift binary is made.

### Actual
`make: *** No rule to make target 'build'.  Stop.`

### Note
Change `make build` to `make`?

--- a/docs/source/contributing/developing.adoc
+++ b/docs/source/contributing/developing.adoc
@@ -230,7 +230,7 @@ For example, *proxy.feature* or *cmd-version.feature*.
 [[running-integration-tests]]
 ==== Running Integration Tests
 
-By default, tests are run against the binary created by `make build` (*_$GOPATH/bin/minishift_*).
+By default, tests are run against the binary created by `make` (*_$GOPATH/bin/minishift_*).
 To run the quick test, use the following command:
 
 ----

